examples/fetch_ic/test/tests.ts

```diff
@@ -0,0 +1,318 @@
+// TODO allow running these tests with and without the authorization header
+// TODO these seem like really good tests for the server and normal fetch too
+// TODO consider how we'll do the axios tests as well
+
+import * as dns from 'node:dns';
+dns.setDefaultResultOrder('ipv4first');
+
+import { Principal } from '@dfinity/principal'; // TODO get rid of this dependency if we installed it
+import { Test, getCanisterOrigin } from 'azle/test';
+import puppeteer, { Browser, Page } from 'puppeteer';
+
+export function getTests(canisterName: string): Test[] {
```

Comment:
I'd like to see a test for a fetch with a Request Object

Reply:
We decided not to implement Request for the time being